cmd/tailscaled/tailscaled.go

```diff
@@ -480,6 +481,15 @@ func startIPNServer(ctx context.Context, logf logger.Logf, logID logid.PublicID,
 		lb, err := getLocalBackend(ctx, logf, logID, sys)
 		if err == nil {
 			logf("got LocalBackend in %v", time.Since(t0).Round(time.Millisecond))
+			if lb.Prefs().Valid() {
+				if err := lb.Start(ipn.Options{}); err != nil {
```

Comment:
line 503 doesn't block for this goroutine to end right?

Reply:
so then 504 would never run?

Reply:
oh wait Run never returns. nvm

Reply:
line 501 blocks until the cancel after the error store:

```
					lbErr.Store(err)
					cancel()
```

So we store the error, then cancel, unblocking Run, then Run checks that atomic error.